### PR TITLE
WPF/UIAutomationTypes:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool

### PR DESCRIPTION
**Pull request by ryalanms**
WPF/UIAutomationTypes:  Generate XML files from most recent RC1 WPF source and update docs with DocsPortingTool.

The RC1 source for these assemblies has 'triple-slash' comments not in the documentation:

*PresentationBuildTasks
PresentationCore
ReachFramework
System.Windows.Controls.Ribbon
System.Windows.Presentation
UIAutomationClient
UIAutomationTypes
WindowsBase*

There is one documentation update PR per assembly. 

(Hide whitespace changes when reviewing files.) 

/cc @dotnet/wpf-developers @carlossanlop

### CODE REVIEW

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 1]
```diff
@@ -1,133 +1,133 @@
...
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Automation.AutomationNotificationProcessing</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Returns the order in which to process the notification.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:127]:
```suggestion
        <summary>Gets the order in which to process the notification.</summary>

```

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 2]
```diff
@@ -1,133 +1,133 @@
...
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Automation.AutomationNotificationKind</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Returns the type of the notification.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:106]:
```suggestion
        <summary>Gets the type of the notification.</summary>

```

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 3]
```diff
@@ -1,133 +1,133 @@
...
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.String</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Returns the the display string of the notification.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:85]:
```suggestion
        <summary>Gets the display string of the notification.</summary>

```

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 4]
```diff
@@ -1,133 +1,133 @@
...
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.String</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Returns the activity ID string of the notification.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:64]:
```suggestion
        <summary>Gets the activity ID of the notification.</summary>

```

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 5]
```diff
@@ -1,133 +1,133 @@
...
+        <Parameter Name="notificationProcessing" Type="System.Windows.Automation.AutomationNotificationProcessing" />
+        <Parameter Name="displayString" Type="System.String" />
+        <Parameter Name="activityId" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="notificationKind">Specifies the type of the notification.</param>
+        <param name="notificationProcessing">Specifies the order in which to process the notification.</param>
+        <param name="displayString">A display string describing the event.</param>
+        <param name="activityId">A unique non-localized string to identify an action or group of actions. Use this to pass additional information to the event handler.</param>
+        <summary>Constructor for notification event args.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:44]:
```suggestion
        <summary>Initializes an instance of the <see cref="System.Windows.Automation.T:NotificationEventArgs"/> class with the specified parameters.</summary>

```

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 6]
```diff
@@ -1,133 +1,133 @@
...
+  <AssemblyInfo>
+    <AssemblyName>UIAutomationTypes</AssemblyName>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Windows.Automation.AutomationEventArgs</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>NotificationEventArgs event args class</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:17]: @ryalanms this isn't very helpful for the user. Can you please provide a better description?

File: xml/System.Windows.Automation/AutomationIdentifier.xml [hunk 7]
```diff
@@ -1,208 +1,217 @@
...
+      <Docs>
+        <summary>Gets the underlying numerical identifier.</summary>
+        <value>The underlying identifier.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Use LookupById method to convert back from Id to an
+            AutomationIdentifier
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/AutomationIdentifier.xml:162]: Is this information useful for the final user?
```suggestion
Use the `LookupById` method to convert back from `Id` to an `AutomationIdentifier`.

```

File: xml/System.Windows.Automation/AutomationNotificationProcessing.xml [hunk 8]
```diff
@@ -1,121 +1,129 @@
...
+  <TypeSignature Language="C++ CLI" Value="public enum class AutomationNotificationProcessing" />
+  <AssemblyInfo>
+    <AssemblyName>UIAutomationTypes</AssemblyName>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Enum</BaseTypeName>
+  </Base>
+  <Docs>
+    <summary>Specifies the order in which to process a notification when calling AutomationPeer.RaiseNotificationEvent</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/AutomationNotificationProcessing.xml:16]:
```suggestion
    <summary>Specifies the order in which to process a notification when calling <see cref="M:System.Windows.Automation.Peers.AutomationPeer.RaiseNotificationEvent(System.Windows.Automation.AutomationNotificationKind,System.Windows.Automation.AutomationNotificationProcessing,System.String,System.String)"/></summary>

```

File: xml/System.Windows.Automation/AutomationNotificationKind.xml [hunk 9]
```diff
@@ -1,121 +1,121 @@
...
+  <TypeSignature Language="C++ CLI" Value="public enum class AutomationNotificationKind" />
+  <AssemblyInfo>
+    <AssemblyName>UIAutomationTypes</AssemblyName>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Enum</BaseTypeName>
+  </Base>
+  <Docs>
+    <summary>Indicates the type of notification when calling AutomationPeer.RaiseNotificationEvent</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/AutomationNotificationKind.xml:16]:
```suggestion
    <summary>Indicates the type of notification when calling <see cref="M:System.Windows.Automation.Peers.AutomationPeer.RaiseNotificationEvent(System.Windows.Automation.AutomationNotificationKind,System.Windows.Automation.AutomationNotificationProcessing,System.String,System.String)"/></summary>

```

File: xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml [hunk 10]
```diff
@@ -1,64 +1,64 @@
...
+  <AssemblyInfo>
+    <AssemblyName>UIAutomationTypes</AssemblyName>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Windows.Automation.AutomationEventArgs</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>ActiveTextPositionChangedEventArgs event args class</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml:17]: Can you please provide a more meaningful description of this class?

File: xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml [hunk 11]
```diff
@@ -1,64 +1,64 @@
...
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="textRangeProvider" Type="System.Windows.Automation.Provider.ITextRangeProvider" />
+      </Parameters>
+      <Docs>
+        <param name="textRangeProvider">To be added.</param>
+        <summary>Constructor for ActiveTextPositionChanged event args.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml:38]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Windows.Automation.ActiveTextPositionChangedEventArgs"/> class.</summary>

```

File: xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml [hunk 12]
```diff
@@ -1,64 +1,64 @@
...
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>UIAutomationTypes</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Automation.Provider.ITextRangeProvider</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Returns the text range where the change occurred, if applicable.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/ActiveTextPositionChangedEventArgs.xml:58]:
```suggestion
        <summary>Gets the text range where the change occurred, if applicable.</summary>

```

**Review by carlossanlop — COMMENTED** [review]
Some suggestions.

File: xml/System.Windows.Automation/NotificationEventArgs.xml [hunk 13]
```diff
@@ -1,133 +1,138 @@
...
+        <Parameter Name="notificationProcessing" Type="System.Windows.Automation.AutomationNotificationProcessing" />
+        <Parameter Name="displayString" Type="System.String" />
+        <Parameter Name="activityId" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="notificationKind">Specifies the type of the notification.</param>
+        <param name="notificationProcessing">Specifies the order in which to process the notification.</param>
+        <param name="displayString">A display string describing the event.</param>
+        <param name="activityId">A unique non-localized string to identify an action or group of actions. Use this to pass additional information to the event handler.</param>
+        <summary>Initializes an instance of the <see cref="System.Windows.Automation.T:NotificationEventArgs"/> class with the specified parameters.</summary>
```
Comment (carlossanlop) [inline xml/System.Windows.Automation/NotificationEventArgs.xml:44]: Typo:
```suggestion
        <summary>Initializes an instance of the <see cref="T:System.Windows.Automation.NotificationEventArgs"/> class with the specified parameters.</summary>

```

**Review by gewarren — APPROVED** [review]

**Pull request merged**